Wrappers/Python/ccpi/web_viewer/trame_viewer.py

```diff
@@ -374,3 +374,20 @@ def create_drawer_ui_elements(self):
         raise NotImplementedError(
             "This function is not implemented in the base class, but you can expect an implementation in it's sub"
             " classes.")
+
+            self.cil_viewer.getRenderer().Render()
```

Comment:
i don't think this should be here

Reply:
Ya know I think the merge broke a couple things, let me clean this up!
```suggestion
```